windows/CodePush.Shared/SettingsManager.cs

```diff
@@ -1,13 +1,21 @@
 ﻿using Newtonsoft.Json;
 using Newtonsoft.Json.Linq;
 using System;
+#if WINDOWS_UWP
 using Windows.Storage;
+#else
+using CodePush.Net46.Adapters.Storage;
+#endif
 
 namespace CodePush.ReactNative
 {
     internal class SettingsManager
     {
+#if WINDOWS_UWP
         private static ApplicationDataContainer Settings = ApplicationData.Current.LocalSettings.CreateContainer(CodePushConstants.CodePushPreferences, ApplicationDataCreateDisposition.Always);
+#else
+        private static ApplicationDataContainer Settings = new ApplicationDataContainer();
```

Comment:
You are right absolutely! Thanks for this catch.
Fixed in 03af9fcf29e95a00dc8dbdee2214d0e4e2ab2bda